### PR TITLE
Update community.json

### PR DESCRIPTION
**Pull request by Julia-Ritter**
Adding FactSet

### CODE REVIEW

File: website/data/community.json [hunk 1]
```diff
@@ -342,5 +342,15 @@
       "type": "application-provider",
       "badges": [],
       "description": "<p>NexJ’s Integrated Advisor Desktop acts as a hub connecting best-of-breed solutions to deliver a superior seamless advisor experience that improves productivity, accelerates client acquisition, increases client retention and maximizes relationships. It contains all information about each client and household and integrates to other systems, allowing advisors to complete processes without logging into multiple applications and switching contexts.</p>"
+    },
+    {
+    "title": "FactSet",
+        "publisher": "FactSet",
+        "image": "/img/users/FactSet.png",
```
Comment (greyseer256) [inline website/data/community.json:349]: Do you need an image for this? I can provide one if needed.

File: website/data/community.json [hunk 2]
```diff
@@ -342,5 +342,15 @@
...
+        "infoLink": "https://www.factset.com",
+        "docsLink": "https://www.factset.com/solutions/data-solutions",
+        "description": "FactSet delivers financial data, analytics, and open technology in a digital platform to help approximately 180,000 users see more, think bigger, and do their best work. We empower the financial community to acquire new insights with access to comprehensive and contextualized content and multiply their effectiveness with flexible next-generation workflow solutions powered by innovative technologies.We meet our clients wherever they work and provide a platform built for seamless interoperability to enable users to create the desktop of their choice.",
```
Comment (greyseer256) [inline website/data/community.json:352]:
```suggestion
        "description": "FactSet delivers financial data, analytics, and open technology in a digital platform to help approximately 180,000 users see more, think bigger, and do their best work. We empower the financial community to acquire new insights with access to comprehensive and contextualized content and multiply their effectiveness with flexible next-generation workflow solutions powered by innovative technologies. We meet our clients wherever they work and provide a platform built for seamless interoperability to enable users to create the desktop of their choice.",
```

File: website/data/community.json (hunk 1 above)
Comment (robmoffat) [reply website/data/community.json:349]: it already exists in the project

**Review by robmoffat — APPROVED** [review]

**Review by mistryvinay — APPROVED** [review]
LGTM

**Review by greyseer256 — APPROVED** [review]

**Pull request merged**